src/components/VmDetails/cards/UtilizationCard/UtilizationCharts/BarChart.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { ChartBar, Chart, ChartAxis, ChartTooltip, ChartStack, ChartLabel } from '@patternfly/react-charts'
+
+const CustomLabel = ({ label, text, ...rest }) => {
+  const t = label ? typeof label === 'function' ? label(rest.datum) : label : ''
+  return (
+    <g>
+      <ChartLabel {...rest} x={50} dy={40} text={t} />
+      <ChartTooltip {...rest} text={text} style={{ fontSize: 18 }} />
+    </g>
+  )
+}
+
+CustomLabel.defaultEvents = ChartTooltip.defaultEvents
+CustomLabel.propTypes = {
+  text: PropTypes.oneOfType([ PropTypes.string, PropTypes.func ]),
+  label: PropTypes.oneOfType([ PropTypes.string, PropTypes.func ]),
+}
+
+const BarChart = ({ data, additionalLabel, thresholdWarning, thresholdError, id, ...rest }) => {
```

Comment:
I added a bunch of filesystems to one of my VMs and the new chart looks like it needs some work:
![screenshot-localhost-3000-2019 07 03-17-06-16](https://user-images.githubusercontent.com/3985964/60625116-e13c2480-9db4-11e9-81ac-3c825b2d354e.png)

The old view of the same VM:
![screenshot-chopper local dickerson-web com-2019 07 03-17-06-43](https://user-images.githubusercontent.com/3985964/60625146-f0bb6d80-9db4-11e9-887f-1b0c10ddf80b.png)


Maybe use the [`Progress`](https://patternfly-react.surge.sh/patternfly-4/components/progress/) component like you used `UtilizationBar` previously?

Reply:
Would this be easier than stacking charts? 
https://www.patternfly.org/v4/documentation/react/components/progress/